usaspending_api/common/api_request_utils.py

```diff
@@ -464,12 +466,16 @@ def build_response(self):
                     "matched_ids": Location.objects.filter(Q(**{"location_congressional_code": row[0], "location_state_code": row[1], "location_state_name": row[2]})).values_list("location_id", flat=True)
                 }
                 response_object.append(response_row)
+                if len(response_object) >= limit:
```

Comment:
My bad, I didn't realize we could use `distinct()` on a values list. We now sort and limit in the query, but this line is still needed since we _may_ not fill our response list in a single query. (I.E. a search for "WASHINGTON" would give us Washington state, but we should cascade down and fill the remaining 8 slots with cities named "Washington")